xbmc/platform/linux/threads/ThreadImplLinux.cpp

```diff
@@ -129,43 +83,23 @@ void CThreadImplLinux::SetThreadInfo(const std::string& name)
   pthread_setname_np(m_handle, name.c_str());
 #endif
 
-  // get user max prio
-  const int maxPrio = ThreadPriorityToNativePriority(ThreadPriority::HIGHEST);
-  const int userMaxPrio = GetUserMaxPriority(maxPrio);
-
-  // if the user does not have an entry in limits.conf the following
-  // call will fail
-  if (userMaxPrio > 0)
-  {
-    // start thread with nice level of application
-    const int appNice = getpriority(PRIO_PROCESS, getpid());
-    if (setpriority(PRIO_PROCESS, m_threadID, appNice) != 0)
-      CLog::Log(LOGERROR, "[threads] failed to set priority: {}", strerror(errno));
-  }
+  m_name = name;
 }
 
 bool CThreadImplLinux::SetPriority(const ThreadPriority& priority)
 {
-  // keep priority in bounds
+  std::call_once(flag,
+                 [this]() { CLog::Log(LOGDEBUG, "[threads] app priority: '{}'", s_appPriority); });
```

Comment:
```suggestion
                 []() { CLog::Log(LOGDEBUG, "[threads] app priority: '{}'", s_appPriority); });
```
Clang complains about the unused capture:
```
.../xbmc/platform/linux/threads/ThreadImplLinux.cpp:92:19: warning: lambda capture 'this' is not used [-Wunused-lambda-capture]
                 [this]() { CLog::Log(LOGDEBUG, "[threads] app priority: '{}'", s_appPriority); });
                  ^~~~
```

Reply:
Thanks!